From: Etienne Cordonnier <ecordonnier@snap.com>
Subject: [PATCH] chartype.h: don't include stdc-predef.h on Apple

stdc-predef.h is a glibc header which doesn't exist in the macOS SDK
or the Darwin GCC toolchain.

Upstream-Status: Pending
Signed-off-by: Etienne Cordonnier <ecordonnier@snap.com>
---
--- a/src/chartype.h
+++ b/src/chartype.h
@@ -29,7 +29,9 @@
 #ifndef _h_chartype_f
 #define _h_chartype_f
 
+#ifndef __APPLE__
 #include <stdc-predef.h>
+#endif
 /* Ideally we should also test the value of the define to see if it
  * supports non-BMP code points without requiring UTF-16, but nothing
  * seems to actually advertise this properly, despite Unicode 3.1 having
